File: CS/FindTextUsingShortcut/Form1.cs
using DevExpress.Pdf;
using DevExpress.XtraPdfViewer;
using System;
using System.Windows.Forms;

namespace FindTextUsingShortcut {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            pdfViewer1.LoadDocument(@"..\..\Report.pdf");
        }

        private void pdfViewer1_KeyDown(object sender, KeyEventArgs e) {
            if (e.KeyData == Keys.F3) {
                PdfTextSearchParameters parameters = new PdfTextSearchParameters();
                PdfFindDialogOptions options = pdfViewer1.FindDialogOptions;
                parameters.CaseSensitive = options.CaseSensitive;
                parameters.WholeWords = options.WholeWords;
                pdfViewer1.FindText(options.Text, parameters);
            }
        }
    }
}

File: VB/FindTextUsingShortcut/Form1.vb
Imports DevExpress.Pdf
Imports DevExpress.XtraPdfViewer
Imports System
Imports System.Windows.Forms

Namespace FindTextUsingShortcut
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            pdfViewer1.LoadDocument("..\..\Report.pdf")
        End Sub

        Private Sub pdfViewer1_KeyDown(ByVal sender As Object, ByVal e As KeyEventArgs) Handles pdfViewer1.KeyDown
            If e.KeyData = Keys.F3 Then
                Dim parameters As New PdfTextSearchParameters()
                Dim options As PdfFindDialogOptions = pdfViewer1.FindDialogOptions
                parameters.CaseSensitive = options.CaseSensitive
                parameters.WholeWords = options.WholeWords
                pdfViewer1.FindText(options.Text, parameters)
            End If
        End Sub
    End Class
End Namespace
